src/control/RotateControl.ts

```diff
@@ -248,7 +261,10 @@ class RotateControl extends Component<ControlEvents> implements CameraControl, O
     window.addEventListener(BROWSER.EVENTS.MOUSE_MOVE, this._onMouseMove, false);
     window.addEventListener(BROWSER.EVENTS.MOUSE_UP, this._onMouseUp, false);
 
-    this.trigger(CONTROL_EVENTS.HOLD);
+    this.trigger(CONTROL_EVENTS.HOLD, {
+      inputType: INPUT_TYPE.ROTATE,
```

Comment:
inputType has a slightly higher meaning, such as 'touch' and 'mouse'. Or it may feel like a text input method.
What about gestureType?

Reply:
That makes sense, but I would prefer just `type` for the event key in that case.
I'd like to change `INPUT_TYPE` to `GESTURE_TYPE` though :)